### PR TITLE
Fix lots of warnings

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: none
affects: none
release note: none

### CODE REVIEW

File: Canvas/TechDebt/Assignments/CKRichTextInputView.m [hunk 1]
```diff
@@ -376,7 +376,7 @@ - (NSString *)replaceImagesWithThumbnailsInHTML:(NSString *)html
                 [self.webView stringByEvaluatingJavaScriptFromString:js];
             }
             
-            [loadingImages removeObjectForKey:activityImageTag];
+            [self->loadingImages removeObjectForKey:activityImageTag];
```
Comment (bluwave) [inline Canvas/TechDebt/Assignments/CKRichTextInputView.m]: will `self.loadingImages` work here?

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 95.14% | 95.16% | -0.01%
**Student** | 92.69% | 92.69% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.7% | 92.7% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Canvas/TechDebt/Assignments/CKRichTextInputView.m (hunk 1 above)
Comment (vanwagonet) [reply Canvas/TechDebt/Assignments/CKRichTextInputView.m]: It's not declared as a property. `self.` causes a compile error.

**Comment by vanwagonet** [comment]
(rebased)

**Pull request merged**